.travis.yml

```diff
@@ -42,15 +40,26 @@ services: mongodb
 before_install:
     - |
       # General configuration
+      set -e
```

Comment:
I think this will impact the Travis build script, which might break it, as commands put in the `.travis.yml` (rather than in external script) are placed directly in the generated build script

Reply:
this will hurt on failure, not on success (`set -e` only impacts failures), as it would make the script end without the Travis logic running after the failed command to end things gracefully.

Reply:
This issue `set -e` prevents is dealing with exit codes in the "for" loops. This is much simpler this way.